app/instance-initializers/axe-component.js

```diff
@@ -138,12 +148,16 @@ export function initialize() {
      */
     audit() {
       if (this.get('tagName') !== '') {
-
-        axe.a11yCheck(this.$(), this.axeOptions, (results) => {
+        this.get('concurrentAxe').run(this.element, this.axeOptions, (error, results) => {
           if (this.get('isDestroyed')) {
             return;
           }
 
+          if (error) {
+            Ember.Logger.error(error);
```

Comment:
Based on the axe-core API docs, we expect `error` to be uncommon (and if it isn't we are instructed to report issues to axe-core itself). Since this is an exceptional case, can you update this to `throw error` instead of logging (and then swallowing) the error?

Reply:
Yes, makes sense. Should be fixed now.